examples/sprites/src/lib.rs

```diff
@@ -31,8 +31,7 @@ pub fn translate(p: &Point, v: Vector) -> Point {
 // and which can move about over time.
 #[derive(Debug)]
 pub struct Sprite {
-    // All interfaces must be `Send+Sync`, and our naive `Point` struct
-    // isn't, so we wrap it in a `RwLock` to make it so.
+    // We must use interior mutability for managing mutable state, hence the `RwLock`.
```

Comment:
Some of our comments here were misleading - you can't actually use an `RwLock` to make something `Sync` if it isn't already `Sync`, as the user in #1048 found out. The thing that the `RwLock` is actually giving us in this case is *mutability* and I've changed the comments to reflect that.

Reply:
yeah, that's very interesting and took me by surprise!